internal/evaluator/conftest_evaluator.go

```diff
@@ -279,6 +279,18 @@ func (c conftestEvaluator) Evaluate(ctx context.Context, inputs []string) (Check
 	return results, nil
 }
 
+// Helper function to sort Result lists.
+func SortResults(i, j int, slice []output.Result) bool {
```

Comment:
For a function named `SortResults` I was expecting a signature like:

```suggestion
func SortResults(results []output.Result) []output.Result {
```

Reply:
Yeah, maybe this one could be named `ResultCompare` or something like that. And the comment could be changes to be more like `// ResultCompare is for use with sort.Slice to define the sort order`. Or, you could refactor and write a different SortResults that does have the signature Zoran was expecting.

Reply:
I've done the latter - I think it's cleaner code anyway. The interface should match @zregvart's proposal while encapsulating the same sort logic as was there previously.